docs/source/widgets.rst

```diff
@@ -150,8 +150,10 @@ Provides count of new and subject of last e-mail on Gmail.
 Supported platform: platform independent, requiring ``curl``.
 
 This widget expects login information in your ``~/.netrc`` file, e.g.
-``machine mail.google.com login user password pass`` and you have to disable
-`two step verification`_.  `Allow access for less secure apps`_ afterwards.
+``machine mail.google.com login user password pass``. If you have 2FA
+account, you can use 'App password' which is created on
+`Google Security page`_. Otherwise, you have to disable
+`two step verification`_ and `allow access for less secure apps`_.
```

Comment:
I think 2FA and two step verification mean the same thing.  Sorry for introducing the confusion in the linked issue :sweat_smile: